_posts/2023/05/2023-05-11-using-statiq-dotnet-static-app.md

```diff
@@ -70,4 +70,4 @@ namespace SiteName
 ## Summary
 We're now ready to begin expanding on this and building in a theme and adding content. Be sure to continue following my blog for more posts related to Statiq.
 
-{% include link-thumbnail.html path="2023/05/11/statiq-web.jpg" alt="Statiq Web" url="https://jasong.us/3VTUIYj" %}
\ No newline at end of file
+{% include link-thumbnail.html path="2023/05/11/statiq-web.jpg" alt="Statiq Web" url="https://jasong.us/3VTUIYj" %}
```

Comment:
Hmm...not sure what's going on here. I made the change in the GitHub UI so maybe it added/removed trailing whitespace? Looks like the line itself is equivalent.

Reply:
Probably fixed it by the time you got to it. 👍🏽